include/sdk/expHeap.hh

```diff
@@ -31,23 +55,38 @@ struct MEMiExpBlockHead
             u16 alignment : 7;
             u16 groupId : 8;
         } fields;
-    } attribute;
-    u32 size;
-    MEMiExpBlockLink link;
+    } mAttribute;
+    u32 mSize;
+    MEMiExpBlockLink mLink;
 };
 
-struct MEMiExpHeapHead : MEMiHeapHead
+class MEMiExpHeapHead : public MEMiHeapHead
 {
-    MEMiExpBlockList freeBlocks;
-    MEMiExpBlockList usedBlocks;
-    u16 groupId;
-    u16 attribute;
-};
+private:
```

Comment:
Is the `private` -> `public` -> `private` intentional? I kinda like this... This way we see the ctor and dtor first as we would typically expect, yet we can see the change in the access specifier.

Reply:
Yes, it is.